fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaHomeFragment.java

```diff
@@ -64,28 +59,10 @@ private void setupSendButton() {
                 SocialMediaSendFragment::newInstance));
   }
 
-  private void setupSwipeRefresh() {
-    SwipeRefreshLayout swipeRefreshLayout = mSocialMediaHomeFragBinding.swipeRefreshChirps;
-    swipeRefreshLayout.setOnRefreshListener(
-        () -> {
-          mChirpListAdapter.replaceList(
-              mSocialMediaViewModel.getChirpList(mSocialMediaViewModel.getLaoId()));
-
-          final Handler handler = new Handler(Looper.getMainLooper());
-          handler.postDelayed(
-              () -> {
-                if (swipeRefreshLayout.isRefreshing()) {
-                  swipeRefreshLayout.setRefreshing(false);
-                }
-              },
-              1000);
-        });
-  }
-
```

Comment:
While I'm very glad that this type of updating can go :), I think you should remove the `swipeRefreshLayout` from `social_media_home_fragment` as well.

Reply:
You're right 😄